proto/buf.lock

```diff
@@ -8,12 +8,12 @@ deps:
   - remote: buf.build
     owner: cosmos
     repository: cosmos-sdk
-    commit: 3db8b415c96b4b0d85ec0a54ac5701d3
+    commit: 1d1523497b394c50ac17605136043cbc
```

Comment:
These are buf mod updates that has been made due to cosmos sdk dependency update